packages/flutter_tools/lib/src/project.dart

```diff
@@ -291,7 +292,7 @@ class FlutterProject {
     bool macOSPlatform = false,
     bool windowsPlatform = false,
     bool webPlatform = false,
-    bool windowsUwpPlatform = false,
+    bool winuwpPlatform = false,
```

Comment:
nit: this should probably be `winUwpPlatform`. (I'm assuming based on `macOSPlatform` above that the capitalisation isn't specifically to mirror commandline platform arguments)

Reply:
Done (well, there are a few more places that use winuwp but we'll get them in time)